### PR TITLE
start regression testing of test case with VAPPARS keyword 

### PR DESCRIPTION
**Pull request by tskille**
OPM Flow support of keyword VAPPARS used in Solution section was achieve with pr

   https://github.com/OPM/opm-common/pull/3550 

This has now been tested and simulation results are excellent and with this PRI would like to start regression testing.

### CODE REVIEW

File: regressionTests.cmake [hunk 1]
```diff
@@ -1155,3 +1155,10 @@ add_test_compareECLFiles(CASENAME 02_wgrupcon
                          ABS_TOL ${abs_tol}
                          REL_TOL ${rel_tol}
                          DIR wgrupcon)
+add_test_compareECLFiles(CASENAME 01_vappars
+                         FILENAME 01-VAPPARS
```
Comment (bska) [inline regressionTests.cmake:1159]: I don't see a case named `01-VAPPARS`, but we do have a case named [`VAPPARS-01`](https://github.com/OPM/opm-tests/blob/e29c0fa7cac292fedf0f3be49c606c210f5007c4/vappars/VAPPARS-01.DATA) in the `vappars` directory of OPM-Tests.  Did you mean that one?

Comment (tskille) [reply regressionTests.cmake]: thanks @bska, my bad. This is fixed now.

**Comment by bska** [comment]
jenkins build this update_data please

**Comment by bska** [comment]
jenkins build this opm-tests=982 please

**Comment by bska** [comment]
The new reference solutions have been installed on the CI system.  Merging this now to activate the regression test.

**Pull request merged**